docs/migration_guides/algorithms_migration.rst

```diff
@@ -80,8 +80,8 @@ four different primitive implementations, depending on how you want to configure
 
     a. Using **local** statevector simulators for quick prototyping: **Reference Primitives** in :mod:`qiskit.primitives`
     b. Using **local** Aer simulators for finer algorithm tuning: **Aer Primitives** in :mod:`qiskit_aer.primitives`
-    c. Accessing backends using the **Qiskit Runtime Service**: **Runtime Primitives** in :mod:`qiskit_ibm_runtime`
-    d. Accessing backends using a **non-Runtime-enabled provider**: **Backend Primitives** in :mod:`qiskit.primitives`
+    c. Accessing backends using the **IBM's Qiskit Runtime service**: **Runtime primitives** in :mod:`qiskit_ibm_runtime`
+    d. Accessing backends using a **non-native primitves in other provider**: **Backend Primitives** in :mod:`qiskit.primitives`
```

Comment:
I am not sure if this conveys well what the backend primitive does, maybe I don't understand it well. In this case, the backend primitives are the non-native primitive, right? but you are not using a "non-native primitive in other provider", unless you consider qiskit-terra a provider? In any case, the sentence looks a bit difficult to read, what about rephrasing what you wrote in the opflow migration section:

```suggestion
    d. Using a non-native primitive to access backends without a native primitive support: **Backend Primitives** in :mod:`qiskit.primitives`
```

Reply:
As little as I like "backend primitives", I cannot find a better short description. So let's stay with it.